src/UglyToad.PdfPig/Geometry/PdfPath.cs

```diff
@@ -5,27 +5,29 @@ namespace UglyToad.PdfPig.Geometry
     using System.Collections.Generic;
     using System.Linq;
     using System.Text;
+    using UglyToad.PdfPig.Core;
 
     /// <summary>
     /// A path in a PDF document, used by glyphs and page content.
     /// </summary>
     public class PdfPath
     {
-        private readonly List<IPathCommand> commands = new List<IPathCommand>();
+        public readonly List<IPathCommand> Commands = new List<IPathCommand>();
         private PdfPoint? currentPosition;
+        internal TransformationMatrix CurrentTransformationMatrix;
```

Comment:
This is a good addition 👍 

I'd make this a constructor parameter rather than an `internal` settable field. As far as possible in the code I try to avoid publically mutable fields/properties. It makes the code easier to reason about if it can't be changed from elsewhere.